nodes.go

```diff
@@ -245,6 +336,20 @@ func (n *Node) Trash() (*http.Response, error) {
 
 }
 
+func (n *Node) Restore() (*http.Response, error) {
```

Comment:
This also returns a Node which we should return to the user.

Needs a comment as it is an externally visible function.